.readthedocs.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.11"
```

Comment:
Just double checking, is 3.11 ok for those old versions?

Reply:
Good point, I think we should at least verify that our oldest branch at RTD, 3.3, works with 3.11 (I think 3.6 was used back then).

Reply:
It looks like it works here. The backport will tell us if it works on 3.3 as well.

![image](https://github.com/crate/crate/assets/453543/7793d0be-b5fd-42de-9600-7e2276f05413)

Reply:
Because it will need manual pushing to get this past failing CI steps, I need your support to bring it in, @seut.

Reply:
Looks like we need to backport https://github.com/crate/crate/commit/122620fc359080992ee14dca7180b2427a695b63 first

Reply:
There is currently a failure:
```
File "/var/lib/jenkins/workspace/CrateDB/crate_on_pr/blackbox/docs/general/ddl/data-types.rst", line 2207, in data-types.rst
Failed example:
    cmd.process('INSERT INTO my_table (\n    title,\n    protagonist\n) VALUES (\n    \'Alice in Wonderland\',\n    {\n        "age" = \'10\',\n    }\n)')Expected:
    SQLParseException[line 8:5: no viable alternative at input 'VALUES (\n    'Alice in Wonderland',\n    {\n        "age" = '10',\n    }']
Got:
    SQLParseException[line 8:1: no viable alternative at input 'VALUES (\n'Alice in Wonderland',\n{\n"age" = '10',\n}']
```

Reply:
> Looks like we need to backport [122620f](https://github.com/crate/crate/commit/122620fc359080992ee14dca7180b2427a695b63) first

I'll do it now, thx!

Reply:
> There is currently a failure:
> 
> ```
> File "/var/lib/jenkins/workspace/CrateDB/crate_on_pr/blackbox/docs/general/ddl/data-types.rst", line 2207, in data-types.rst
> Failed example:
>     cmd.process('INSERT INTO my_table (\n    title,\n    protagonist\n) VALUES (\n    \'Alice in Wonderland\',\n    {\n        "age" = \'10\',\n    }\n)')Expected:
>     SQLParseException[line 8:5: no viable alternative at input 'VALUES (\n    'Alice in Wonderland',\n    {\n        "age" = '10',\n    }']
> Got:
>     SQLParseException[line 8:1: no viable alternative at input 'VALUES (\n'Alice in Wonderland',\n{\n"age" = '10',\n}']
> ```

Pushed a commit to fix this, it has been addressed for quite some time now by: https://github.com/crate/crate/pull/13886